javascriptv3/example_code/cross-services/textract-react/README.md

```diff
@@ -0,0 +1,211 @@
+# Amazon Textract React example
+
+## Purpose
+
+Shows how to use the AWS SDK for JavaScript, Version 3, to build a React application
```

Comment:
Although it's implicit in this paragraphs, might be useful to add a high level description of Textract - e.g. 'Amazon Textract is a machine learning service that automatically extracts text, handwriting and data from scanned documents to identify, understand, and extract data from forms and tables.'

Reply:
Sounds good, I added it.